tests/wicked/before_test.pm

```diff
@@ -59,6 +59,9 @@ sub run {
         assert_script_run("rcwickedd restart");
         record_info('INFO', 'Setup OpenVPN');
         zypper_call('--quiet in openvpn', timeout => 200);
+        record_info('INFO', 'Setup OVS');
+        zypper_call('--quiet in openvswitch', timeout => 200);
```

Comment:
startandstop suite actually don't need that , please wrap it to additional if

Reply:
whole section of course

Reply:
done